### PR TITLE
display how much comments there are for an entry

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #1713
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -98,6 +98,7 @@ Toggle favorite: 'Marquer comme favori'
 Delete: 'Supprimer'
 "{0} There is no entry.|{1} There is one entry.|]1,Inf[ There are %count% entries.": "{0} Il n'y a pas d'articles.|{1} Il y a un article.|]1,Inf[ Il y a %count% articles."
 http://website: "http://siteweb"
+"{0} No annotations|{1} One annotation|]1,Inf[ %nbComments% annotations": "{0} Aucun commentaire|{1} Un commentaire|]1,Inf[ %nbComments% commentaires"
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: Are we choosing _commentaire_ over _annotation_ ?

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: @nicosomb has done it so far. One or the other doesn't bothers me.

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: I'm prefer _annotation_.

**Comment by j0k3r** [comment]
BTW, from which branch did you start your branch? Because it seems that you started from the master branch (and get all commits from #1653)

**Pull request closed without merging**